.evergreen/config.yml

```diff
@@ -231,9 +231,10 @@ functions:
           SWIFT_VERSION=${SWIFT_VERSION} \
             bash ${PROJECT_DIRECTORY}/.evergreen/install-tools.sh sourcery
           make linuxmain SOURCERY=${PROJECT_DIRECTORY}/opt/sourcery/bin/sourcery
-          git diff --exit-code Tests/LinuxMain.swift
+          # use regex to ignore lines starting with a /
```

Comment:
this and related sourcery changes cherry-picked in from main: dbc65f33a073d37c049de66a46c039979e334786